src/ByteSizeLib/ByteSize.cs

```diff
@@ -142,37 +146,72 @@ public ByteSize(double bytes)
             Bytes = bytes;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the ByteSize structure to the specified
+        /// number of units.
+        /// </summary>
+        /// <param name="value">Number of bits.</param>
         public static ByteSize FromBits(long value)
         {
             return new ByteSize(value);
         }
 
+        /// <inheritdoc cref="FromBits" />
+        /// <param name="value">Number of bytes.</param>
         public static ByteSize FromBytes(double value)
         {
             return new ByteSize(value);
         }
 
         /// <summary>
-        /// Converts the value of the current object to a string.
-        /// The prefix symbol (bit, byte, kilo, mebi, gibi, tebi) used is the
+        /// Converts the value of the current object to a decimal byte string.
+        /// The prefix symbol (bit, byte, kilo, mega, etc.) used is the
         /// largest prefix such that the corresponding value is greater than or
         /// equal to one.
+        /// Use <see cref="ByteSize.ToBinaryString()"/> for binary string representation.
         /// </summary>
         public override string ToString()
         {
             return this.ToString("0.##", CultureInfo.CurrentCulture);
         }
 
+        /// <summary>
+        /// Converts the value of the current object to a decimal byte string.
+        /// The prefix symbol (bit, byte, kilo, mega, etc.) used is the
+        /// largest prefix such that the corresponding value is greater than or
+        /// equal to one.
+        /// Use <see cref="ByteSize.ToBinaryString()"/> for binary string representation.
+        /// </summary>
+        /// <param name="format">A numeric format string.</param>
```

Comment:
This was tough to document, but since under the hood it uses `Double.ToString(format, provider)`, I used the same documentation as what's specified on that for the `format` parameter.

![image](https://user-images.githubusercontent.com/196368/140437249-39a3bbdd-016e-4468-b855-1dd7f21b49c9.png)

Reply:
That's fair, but you have to keep in mind this being in a `double` type has a lot more context to it right out of the gate. `ByteSize` is not a primitive number, so that's why it is a bit too broad IMHO.

I don't necessarily have a "better" suggestion here, was just pointing out from a consumer perspective that it could be a bit too general and would potentially leave people guessing as to how it is supposed to contain.

However, in the end of the day (considering the fact that the library didn't even expose documentation before), this is probably not that big of a deal.